src/m365/spo/commands/site/site-set.ts

```diff
@@ -60,32 +68,48 @@ class SpoSiteSetCommand extends SpoCommand {
     telemetryProps.siteDesignId = typeof args.options.siteDesignId !== undefined;
     telemetryProps.sharingCapabilities = args.options.sharingCapability;
     telemetryProps.siteLogoUrl = typeof args.options.siteLogoUrl !== 'undefined';
+    telemetryProps.resourceQuota = args.options.resourceQuota;
+    telemetryProps.resourceQuotaWarningLevel = args.options.resourceQuotaWarningLevel;
+    telemetryProps.storageQuota = args.options.storageQuota;
+    telemetryProps.storageQuotaWarningLevel = args.options.storageQuotaWarningLevel;
+    telemetryProps.allowSelfServiceUpgrade = args.options.allowSelfServiceUpgrade;
+    telemetryProps.lockState = args.options.lockState;
+    telemetryProps.noScriptSite = args.options.noScriptSite;
     return telemetryProps;
```

Comment:
not sure but if wait is a flag it is also optional and should be in telemetryProps as well.
what do you think ? 🤔

Reply:
Sharp!